server/memcached.js

```diff
@@ -33,3 +32,5 @@ function chimeraData(res){
   });
 }
 
+module.exports.memcachedOperations = memcachedOperations;
+
```

Comment:
the whitespace matters because it makes it hard for people to collaborate if they don't use the same whitespace rules, you get lots of unnecessarily noisy diffs, or have to change your tools settings all the time